src/main/scala/org/ergoplatform/utils/FileUtils.scala

```diff
@@ -1,20 +1,21 @@
 package org.ergoplatform.utils
 
 import java.io.File
+import java.nio.file.Files
+import scala.collection.JavaConverters._
+import scala.util.Try
 
 /**
- * Utilities to work with OS file system
- */
+  * Utilities to work with OS file system
+  */
 object FileUtils {
+
   /**
-	* Perform recursive deletion of directory content.
-	*/
-  def deleteRecursive(dir: File): Unit = {
-    for (file <- dir.listFiles) {
-      if (!file.getName.startsWith(".")) {
-        if (file.isDirectory) deleteRecursive(file)
-        file.delete()
-      }
+    * Perform recursive deletion of directory content.
+    */
+  def deleteRecursive(root: File): Unit = {
+    if (root.exists()) {
+      Files.walk(root.toPath).iterator().asScala.toSeq.reverse.foreach(path => Try(Files.delete(path)))
```

Comment:
Looks like not equivalent change. What about filtering some files with `!file.getName.startsWith(".")`?

Reply:
I also do not know why using Files.walk is better then hand-written recursion (taken in account that height of LevelDB  store tree is very small). Also please remember that I have fixed "Too many open files" issues with java.base/java.nio.file.Files.walkFileTree. It was fixed by using FileUtils.deleteRecursive instead.
But now we are using Files.walk in FileUtils.deleteRecursive. I am not sure that the same problem with delayed closing of files doesn't take place with Files.walk.

Filtering of "." and ".." is not needed because the are not returned by Files.walk iterator.